code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2092,6 +2092,9 @@
 		qdel(O)
 	else if(O.color && istype(O, /obj/item/weapon/paper))
 		O.color = null
+	if(O)
```

Comment:
Fix the logic on this, just remove the istype(paper) check above

Reply:
Fucking old coders